README.dev.md

```diff
@@ -10,7 +10,7 @@ line-endings for text files (source code).
 Disable any ambiguity in your global and current repository settings:
 
 ```shell
-cd /path/to/TulipaBulb.jl
+cd /path/to/TulipaEnergyModel.jl
 git config --unset core.autocrlf         # disable autocrlf in the Bulb repo
```

Comment:
I think there are other occurrences of `Bulb` in the readme (like this one and 3 lines below), maybe do a find-replace?

Reply:
Had only searched for `TulipaBulb`, replaced `Bulb` now too